src/components/List/List.jsx

```diff
@@ -37,15 +74,27 @@ export const List = ({ token }) => {
     setFilterText(e.target.value);
   };
 
+  const setColor = (item) => {
+    if (!item.isActive) {
+      return 'inactive';
+    } else if (item.estimated_next_purchase <= 7) {
```

Comment:
I think this is going to suffer from the same problem as the sorting algorithm. If I add an item as "kinda soon" (14), ten days later it should be shown in the "soon" category, right? It doesn't seem like that will happen here.